### PR TITLE
[VERIFYME] fstab: Split into early and latemount, reserve 128M on /data

### PR DESCRIPTION
**Pull request by ix5**
Makes things neater with the stage-separated mounting logic in Q and later.

### CODE REVIEW

**Comment by Paulbouchara** [comment]
Shouldn't the tag be in yoshino repo ? 😛

**Comment by ix5** [comment]
Oh crap yeah, that happens when you use the gh cli tool...

File: rootdir/vendor/etc/fstab.nile [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/dev/block/bootdevice/by-name/system       /            ext4    ro,barrier=1                                                  wait,recoveryonly,verify,slotselect
```
Comment (pablomh) [inline rootdir/vendor/etc/fstab.nile:5]: Is the removal of 'recoveryonly' intended (same as in following line)?

Comment (ix5) [reply rootdir/vendor/etc/fstab.nile:5]: Yes, it's something that blindly got copied and has no place in recent Android versions.

**Pull request merged**